### PR TITLE
Apply an unelevated card style to stats cards

### LINKED ISSUE #11468: Stats cards bounds cannot be seen on tablets when using the new material theme's background color
This issue was originally raised by @planarvoid [here](https://github.com/wordpress-mobile/WordPress-Android/pull/11432#issuecomment-599545387).

### Actual behavior
| Stats cards on `develop` | Stats cards on `material-theme` |
|---|---|
|develop|material-theme|

### Expected behavior
@mattmiklic has provided a mockup of how this should like:

Cards

### Steps to reproduce the behavior

Using a build from the `material-theme` branch, open the stats screen on a tablet.

### PR DESCRIPTION
**Pull request by renanferrari**
Fixes #11468

This PR creates a new unelevated card style and applies it to the stats cards.

| Phone (Light) | Phone (Dark) |
|---|---|
|phone-light|phone-dark|

| Tablet (Light) | Tablet (Dark) |
|---|---|
|tablet-light|tablet-dark|

## To test

Open the stats screen on a tablet and make sure it looks like the provided mockup (in the linked issue). Test on both light and dark modes.

---

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by iamthomasbishop** [comment]
@renanferreri @mattmiklic in dark theme, do you think the cards should be elevated to 1dp, similar to what’s shown in [this example](https://storage.googleapis.com/spec-host/mio-staging%2Fmio-design%2F1581631970573%2Fassets%2F1xFyDsHomV7-rZvjeZ9bk3BhAN1bNLLAQ%2Fdarktheme-anatomy.png) on the [Material guidelines](https://material.io/design/color/dark-theme.html#properties)? Or are we assuming the cards are on surface level (0dp)?

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/ManagementControlUseCase.kt [hunk 1]
```diff
@@ -41,6 +41,7 @@ class ManagementControlUseCase
                         icon = R.drawable.ic_plus_white_24dp,
                         textResource = R.string.stats_management_add_new_stats_card,
                         navigationAction = Companion.create(this::onClick),
+                        showDivider = false,
```
Comment (planarvoid) [inline WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/ManagementControlUseCase.kt]: good catch 👍

**Comment by osullivanchris** [comment]
@iamthomasbishop @mattmiklic this looks fine to me! But just curious if we go away from cards and elevation would we consider going away from boxes too? If I was designing this page from scratch with no page background colour, I would probably do it with horizontal lines dividing the blocks of content, but not specifically put borders around the boxes, if that makes sense? Its a little bit more work so it could be a later step to be considered.

Here's a quick hack

borders to divs

**Comment by renanferrari** [comment]
>Or are we assuming the cards are on surface level (0dp)?

Hi @iamthomasbishop 👋 Yes, in this case we're assuming the cards are on surface level.

>If I was designing this page from scratch with no page background colour, I would probably do it with horizontal lines dividing the blocks of content, but not specifically put borders around the boxes, if that makes sense? Its a little bit more work so it could be a later step to be considered.

Thank you for your input on this, @osullivanchris! From a development perspective, both options (with and without boxes) are quite simple to achieve, but having the same look on both phones and tablets is even simpler, which is the main reason I leaned towards the option presented here. @mattmiklic and @SylvesterWilmott also considered these options and they may have some thoughts to share about them, from a design perspective.

@planarvoid Thanks a lot for the review! 🙇‍♂️

**Comment by mattmiklic** [comment]
> Or are we assuming the cards are on surface level (0dp)?

> if we go away from cards and elevation would we consider going away from boxes too

What's seen above is consistent with the Material Design rules for <a href="https://material.io/components/cards/">outlined cards</a>. This felt like the most appropriate treatment for this usage, where there are many cards together on the same screen. Outlined cards have 0 elevation. So this is still a standard card treatment, just one that has a little bit more subtle visual style than elevated cards.

Re: @osullivanchris's mockup; that works for single-column views and it's consistent with how we use cards in the Reader and the Blog Posts screen. But since Stats includes a two-column view (on tablets, or on a phone when in landscape) this feels like the better approach to me.

**Comment by iamthomasbishop** [comment]
> What's seen above is consistent with the Material Design rules for outlined cards.

That makes sense, thanks @mattmiklic !

**Comment by osullivanchris** [comment]
Yep sounds good to me too!

**Pull request merged**